boskos/resources.json

```diff
@@ -164,7 +164,19 @@
       "k8s-jkns-upgrade-fixed-1",
       "k8s-jkns-gce-kube-push",
       "k8s-e2e-gce-scalability",
-      "k8s-jkns-e2e-gce-prl-flaky"
+      "k8s-jkns-e2e-gce-prl-flaky",
+      "e2e-gce-gci-qa-m56",
+      "e2e-gce-gci-qa-serial-m56",
+      "e2e-gce-gci-qa-slow-m56",
+      "e2e-gce-gci-qa-m57",
+      "e2e-gce-gci-qa-serial-m57",
+      "e2e-gce-gci-qa-slow-m57",
```

Comment:
ah, I see, I did not see anything from earlier projects, gonna remove those, thanks for the info :-)